src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Configuration/OpenIdConnectConfiguration.cs

```diff
@@ -161,6 +161,18 @@ public OpenIdConnectConfiguration(string json)
         public string EndSessionEndpoint { get; set; }
 
         /// <summary>
+        /// Gets or sets the frontchannel_logout_session_supported.
```

Comment:
Chen, can you do us a favor and modify the comment for EndSessionEndpoint and add underscores to "end session endpoint" => "end_session_endpoint".  then merge into dev.